docs/reference/intro.asciidoc

```diff
@@ -255,13 +201,9 @@ create secondary clusters to serve read requests in geo-proximity to your users.
 the active leader index and handles all write requests. Indices replicated to
 secondary clusters are read-only followers.
 
-[discrete]
-[[admin]]
-==== Care and feeding
```

Comment:
while I hate this heading, I do think this is its own concept

Reply:
ok I'll rename it